cfgov/unprocessed/apps/paying-for-college/js/disclosures/index.js

```diff
@@ -83,6 +84,9 @@ const app = {
             // Update expenses model bases on region and salary
             const region = schoolValues.BLSAverage.substr(0, 2);
             $('#bls-region-select').val(region).change();
+            $step2.removeClass('a-btn__disabled');
+            $step2.text('Continue to Step 2');
```

Comment:
This can be removed?

Reply:
This overwrites the `<svg>` in the button content, so it is necessary, although I can do it in a different way.

Reply:
Ahh maybe reference and remove the SVG directly, for clarity.